app/src/ui/history/commit-list-item.tsx

```diff
@@ -1,14 +1,22 @@
 import * as React from 'react'
 import { Commit } from '../../models/commit'
+import { Repository } from '../../models/repository'
 import { IAvatarUser } from '../../models/avatar'
 import { RichText } from '../lib/rich-text'
 import { Avatar } from '../lib/avatar'
 import { RelativeTime } from '../relative-time'
+import { getDotComAPIEndpoint } from '../../lib/api'
+import { clipboard } from 'electron'
+import { showContextualMenu, IMenuItem } from '../main-process-proxy'
 
 interface ICommitProps {
+  readonly repository: Repository | null
```

Comment:
Oh, I see further down... could this be `gitHubRepository: GitHubRepository | null` instead? That way we don't have the weirdness of having a `null` repository.